docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.5'
+
+services:
+
+  vegeta:
+    build: .
+    image: flaviostutz/vegeta
```

Comment:
Would love to build and publish an official project Docker image. I see you opened a separate issue for that. What would it take to get that done?

Reply:
The Dockerfile/docker-compose.yml created really solves https://github.com/tsenart/vegeta/issues/525. We would close #525 with current PR.

For an official Docker image, you should create a new Repository in Dockerhub with tsenart/vegeta name and we could configure an automated build there that automatically build master "branch" to "latest" and any tagged git to a tagged image in Dockerhub. I have not prepared this Dockerfile thinking about being an official image, because it doesn't support all vegeta properties. We could discuss that in a separate issue if you like.

Reply:
Another option is we can use a random name in "image" field, because this is only a local name (not necessary published in Dockerhub). What do you think?